examples/schema_template_vrf/main.tf

```diff
@@ -11,13 +11,77 @@ provider "mso" {
   password = "" # <MSO pwd>
   url      = "" # <MSO URL>
   insecure = true
+  platform = "nd"
 }
 
-resource "mso_schema_template_vrf" "demovrf01" {
-    schema_id       = "5c4d5bb72700000401f80948"
-    template        ="Temp200"
-	name            = "vrf982"
-	display_name    ="vrf982"
-	layer3_multicast=false
-  
+data "mso_tenant" "tenant_test" {
+  name         = "ansible_test"
+  display_name = "ansible_test"
+}
+
+resource "mso_schema" "schema_test" {
+  name = "terraform_schema"
+  template {
+    name         = "Template1"
+    display_name = "Template1"
+    tenant_id    = data.mso_tenant.tenant_test.id
+  }
+}
+
+resource "mso_schema_template_vrf" "vrf" {
+  schema_id              = mso_schema.schema_test.id
+  template               = tolist(mso_schema.schema_test.template)[0].name
```

Comment:
I would add a comment above the resource to tell the user that we are associating "Template1" here from mso_schema.schema_test.template to be more intuitive